doc/man/maestro-config.1

```diff
@@ -45,6 +45,42 @@ the GitHub Issues page
 
 .SH EXAMPLE
 
+.SS Prompt without existing account information
+
+.PP
+This shows what the prompt will look like when there is no account information located at
+.I ~/.maestro/aws_account_info.json:
+
+.RS
+$ maestro config
+
+Please enter your AWS Account Number:
+
+$ XXXXXXXXXXXX
+
+Please enter the region for your AWS services:
+
+$ us-east-2
+.RE
+
+.SS Prompt with existing account information
+
+.PP
+This shows what the prompt will look like when there is account information located at
+.I ~/.maestro/aws_account_info.json:
+
+.RS
+$ maestro config
+
+Please enter your AWS Account Number [********XXXX]:
+
+$ XXXXXXXXXXXX
+
+Please enter the region for your AWS services [us-east-2]:
+
+$ us-east-2
+.RE
+
```

Comment:
In accordance of man page "**man-pages**(7)" under "**STYLE GUIDE**" > "**Formatting conventions (general)**":

> When showing example shell sessions, user input should be formatted in bold

, we should change the user input to be bold. I can commit that change if you like.

Reply:
Sounds good. Go for it.